prinz/src/main/scala/pl/touk/knussknacker/prinz/enrichers/IrisSpeciesEnricher.scala

```diff
@@ -0,0 +1,23 @@
+package pl.touk.nussknacker.prinz.enrichers
+
+import javax.validation.constraints.NotBlank
+import pl.touk.nussknacker.engine.api.{MetaData, MethodToInvoke, ParamName, Service}
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+
+/**
+  * Computes species for iris based on dimensions. All entries are classified
+  * as Iris setosa.
+  */
+class IrisSpeciesEnricher extends Service {
```

Comment:
I wouldn't use comments in code. Clean code should explain itself. Name it to be expresive

Reply:
btw this name is expresive enough imo

Reply:
Dropped the comment.